File: src/programmersguide/programmingemail/savemessageasdraft/java/SaveMessageAsDraft.java
/* 
 * Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
 *
 * This file is part of Aspose.Email. The source code in this file
 * is only intended as a supplement to the documentation, and is provided
 * "as is", without warranty of any kind, either expressed or implied.
 */
 
package programmersguide.programmingemail.savemessageasdraft.java;

import com.aspose.email.*;

public class SaveMessageAsDraft
{
    public static void main(String[] args) throws Exception
    {
        // The path to the documents directory.
        String dataDir = "src/programmersguide/programmingemail/savemessageasdraft/data/";

        // Create a new instance of MailMessage class
        MailMessage message = new MailMessage();

        // Set sender information
        message.setFrom(new MailAddress("from@domain.com", "Sender Name", false));

        // Add recipients
        message.getTo().add(new MailAddress("to1@domain.com", "Recipient 1", false));
        message.getTo().add(new MailAddress("to2@domain.com", "Recipient 2", false));

        // Set subject of the message
        message.setSubject("New message created by Aspose.Email for Java");

        // Set Html body of the message
        message.setHtmlBody("<b>This line is in bold.</b> <br/> <br/>"
                + "<font color=blue>This line is in blue color</font>");

        // Create an instance of MapiMessage and load the MailMessag instance into it
        MapiMessage mapiMsg = MapiMessage.fromMailMessage(message);

        // Set the MapiMessageFlags as UNSENT and FROMME
        mapiMsg.setMessageFlags(MapiMessageFlags.MSGFLAG_UNSENT | MapiMessageFlags.MSGFLAG_FROMME);

        // Save the MapiMessage to disk
        mapiMsg.save(dataDir + "New-Draft.msg");

        // Display Status.
        System.out.println("Message saved as draft successfully.");
    }
}

File: Plugins/Aspose_Email_Java_for_PHP/src/aspose/email/ProgrammingEmail/SaveMessageAsDraft.php
<?php
namespace Aspose\Email\ProgrammingEmail;

use com\aspose\email\MailMessage as MailMessage;
use com\aspose\email\MailAddress as MailAddress;
use com\aspose\email\MapiMessage as MapiMessage;
use com\aspose\email\MapiMessageFlags as MapiMessageFlags;
class SaveMessageAsDraft{

    public static function run($dataDir=null){
        # Create a new instance of MailMessage class
        $message = new MailMessage();

            # Set subject of the message
        $message->setSubject("New message created by Aspose.Email for Java");

        $mail_address = new MailAddress();

        # Set Html body
        $message->setHtmlBody("<b>This line is in bold.</b> <br/> <br/>" .
            "<font color=blue>This line is in blue color</font>");

        # Set sender information
        $message->setFrom(new MailAddress("from@domain.com", "Sender Name", false));

        # Add TO recipients
        $message->getTo()->add(new MailAddress("to1@domain.com", "Recipient 1", false));
        $message->getTo()->add(new MailAddress("to2@domain.com", "Recipient 2", false));

        # Create an instance of MapiMessage and load the MailMessag instance into it
        $mapiMessage=new MapiMessage();
        $mapi_msg = $mapiMessage->fromMailMessage($message);

        # Set the MapiMessageFlags as UNSENT and FROMME
        $mapi_message_flags = new MapiMessageFlags();
//        $mapi_msg->setMessageFlags($mapi_message_flags->MSGFLAG_UNSENT || $mapi_message_flags->MSGFLAG_FROMME);

        # Save the MapiMessage to disk
        $mapi_msg->save($dataDir . "New-Draft.msg");

        # Display Status
        print "Draft saved Successfully.".PHP_EOL;
    }

}
?>